scripts/generate-clients/config.js

```diff
@@ -1,7 +1,7 @@
 // Update this commit when taking up new changes from smithy-typescript.
 module.exports = {
   // Use full commit hash as we explicitly fetch it.
-  SMITHY_TS_COMMIT: "05b2fb2f92af20e824cc71ed4490ed0c7075964e",
```

Comment:
The commit `25726c12c40479b73c2919c56e2ce8c41e2a22d6` is from `bump-0.21.1` branch. We need to bump from merged commit in mainline, which is `cf9d6fe3483ba027f98e87d1921c32dd94e2c1f9`

Reply:
The `export type` change was merged in https://github.com/aws/aws-sdk-js-v3/pull/6171